CHANGES.rst

```diff
@@ -7,10 +7,15 @@ New Features
 - Spectral axis can now be any axis, rather than being forced to be last. See docs
   for more details. [#1033]
 
+- Spectrum1D now keeps the entire input GWCS instead of extracting only the spectral
+  information. [#1074]
+
 Other Changes and Additions
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-1.11.0 (unreleased)
+- JWST reader no longer transposes the input data cube for 3D data. [#1074]
```

Comment:
correct me if im wrong, this was just my understanding from our convo earlier - but would a better description be that Spectrum1D now allows GWCS as input, and when using the JWST reader to load a Spectrum1D the full GWCS is now loaded?

Reply:
Good call, I'll update it.

Reply:
Does that sound better?